packages/beacon-push-prize-distribution/src/handler.ts

```diff
@@ -0,0 +1,35 @@
+import { Relayer } from 'defender-relay-client';
+import { DefenderRelayProvider, DefenderRelaySigner } from 'defender-relay-client/lib/ethers';
+import { getContracts, beaconPushPrizeDistribution } from '@pooltogether/v4-autotasks-library';
+import { mainnet, testnet } from '@pooltogether/v4-pool-data';
+
+export async function handler(event: any) {
+  const provider = new DefenderRelayProvider(event);
+  const signer = new DefenderRelaySigner(event, provider, { speed: 'fast' });
+  const relayer = new Relayer(event);
+
+  const chainId = Number(process.env.CHAIN_ID);
+  const contracts = getContracts(chainId, mainnet, testnet);
+
+  try {
+    const transactionPopulated = await beaconPushPrizeDistribution(contracts, {
+      chainId,
+      provider: signer,
+    });
+
+    if (transactionPopulated) {
+      let transactionSentToNetwork = await relayer.sendTransaction({
+        data: transactionPopulated.data,
+        to: transactionPopulated.to,
+        gasLimit: 500000,
+        speed: 'fast',
+      });
+
+      console.log(`TransactionHash: ${transactionSentToNetwork.hash}`);
+    } else {
+      console.log('Error sending transaction');
```

Comment:
Would be nice to have a more explicit error:
```suggestion
      console.log('PrizeDistributionFactoryV2: Transaction not populated');
```